classes/01class/exercises/c01-e04/drii-cavalcanti/c01-e04.txt

```diff
@@ -0,0 +1,15 @@
+Create a S3 bucket:
+S3 -> Create a bucket
+Bucket name: c01-e04-drii-cavalcanti
+Choose: c01-e04-drii-cavalcanti -> Upload -> Add File
+
+Linux Terminal:
+ssh -i "<my key pair>.pem" ubuntu@ec2-18-233-100-104.compute-1.amazonaws.com
+ubuntu@ip-172-31-88-249:sudo apt install awscli | aws configure (using my previous accessKeys)
```

Comment:
Could you try to use Instance Profiles instead of configuring your access keys manually on the instances?

https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use_switch-role-ec2_instance-profiles.html